internal/support/notifications/sending_service.go

```diff
@@ -130,9 +131,40 @@ func sendMail(
 	return tr
 }
 
-func restSend(message string, url string, lc logger.LoggingClient) models.TransmissionRecord {
+func buildSmtpMessage(subject string, contentType string, message string) []byte {
+	// required CRLF at ends of lines and CRLF between header and body for SMTP RFC 822 style email
+	buf := bytes.NewBufferString("Subject: " + subject + "\r\n")
+
+	// only add MIME header if notification content type was set
+	// maybe provide charset overrides as well?
+	if contentType != "" {
+		buf.WriteString(fmt.Sprintf("MIME-version: 1.0;\r\nContent-Type: %s; charset=\"UTF-8\";\r\n", contentType))
+	}
+
+	buf.WriteString("\r\n")
+
+	//maximum line size is 1000, insert newlines in message if needed
+	ln := 998
+	idx := 0
+
+	for len(message) > idx+ln {
```

Comment:
This doesn't take into consideration that `message` could already have newlines appropriately.